<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="clouds_tab">
      <attribute name="title">
       <string>Clouds</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Please connect to your clouds :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="launch_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Launch Demerio</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="account_tab">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,0,1,0,0,0">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Reconstruct your data :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="output_btn">
            <property name="text">
             <string>Select Directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="11,1">
          <item>
           <widget class="QLabel" name="start_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="start_btn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>start</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_version">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
